### PR TITLE
Point Verdant Delivery Points + Long Range Holopad in Cells + Konyang Police Fax + Police Speedloader fix + L + Ratio

### LINKED ISSUE #18562: [Bug]: Point Verdant .45 fast loaders have 30 instead of 6 bullets in them, sprite doesnt change when empty, sprite shows rubber bullets opposed to lethal bullets.
### Checks

- [X] I have searched the bug with a few keywords, and I confirm this bug was not yet reported.

### Location

Away Site

### Describe the issue

Exactly as it says in the title. The fast loaders are a bullet bag of holding, capable of storing far more ammunition than reasonable. Furthermore, they dont change sprites when empty, and also show rubber bullet tips despite the ammo seemingly being lethal.

image

### How to reproduce

1. Spawn as cop
2. Go to the ready room
3. Pick up a fastloader
4. Congrats, you have enough ammunition to conquer a small nation.

### Round ID

_No response_

### Staff ckey

_No response_

### PR DESCRIPTION
**Pull request by Ben10083**
Additional Point Verdant Cargo Delivery Points added
- Hotel
- Up!Burger
- Corporate Docks

fixes #18562

Expanded code for receptacles, from payment modifications to spawn limits, as well as allowing for specific names for designations to better handle away sites with multiple delivery points

### CODE REVIEW

**Comment by Ben10083** [comment]
!review

**Review by NonQueueingMatt — APPROVED** [review]

**Review by DreamySkrell — CHANGES_REQUESTED** [review]
Did you test this in game? Are the different delivery points actually functional? Do they show as different delivery points on the packages? Or can a package be delivered to any point?

**Review by DreamySkrell — CHANGES_REQUESTED** [review]
The packages should show which delivery point they are for, not just by coords. A package to the hotel or whatever should show it is for the hotel, and not for the spaceport (with hotel coords).

This should be done with a var like `override_name` on the delivery point (and if it is null, it should use the default name of the ship or away site).

**Comment by Ben10083** [comment]
image
new functionality for names and payment modifier

File: code/modules/cargo/delivery/package.dm [hunk 1]
```diff
@@ -13,11 +13,13 @@
 
 	var/delivery_point_id = ""
 	var/datum/weakref/delivery_point_sector
+	var/delivery_site = "Unknown"
```
[the remark this answers is not in the document]
Comment (Ben10083) [reply code/modules/cargo/delivery/package.dm:17]: no

[the remark this answers is not in the document]
Comment (DreamySkrell) [reply code/modules/cargo/delivery/package.dm:17]: How does this affect packages outside of konyang? They shouldn't just always say "Unknown" or whatever. This should be documented too.

[the remark this answers is not in the document]
Comment (Ben10083) [reply code/modules/cargo/delivery/package.dm:17]: Added extra documentation for clarification. Receptacle can replace the delivery_site, and it falls back on away site name if override_name isnt set in receptacle.

This is similar to how it works before, minus the override_name option

image

**Review by DreamySkrell — APPROVED** [review]

**Review by alsoandanswer — APPROVED** [review]

**Pull request merged**